signac/contrib/project.py

```diff
@@ -1177,6 +1191,7 @@ def repair(self, fn_statepoints=None, index=None, job_ids=None):
             raise JobsCorruptedError(corrupted)
 
     def _sp_index(self):
+        "Update and return the statepoint index cache"
```

Comment:
For convention's sake (and future proofing against extending the docstring), I would triple quote this.